src/DynamicRupture/FrictionLaws/GpuImpl/SlipLaw.h

```diff
@@ -0,0 +1,49 @@
+#ifndef SEISSOL_GPU_SLIPLAW_H
+#define SEISSOL_GPU_SLIPLAW_H
+
+#include "DynamicRupture/FrictionLaws/GpuImpl/SlowVelocityWeakeningLaw.h"
+
+namespace seissol::dr::friction_law::gpu {
+template <typename TPMethod>
+class SlipLaw : public SlowVelocityWeakeningLaw<SlipLaw<TPMethod>, TPMethod> {
+  public:
+  using SlowVelocityWeakeningLaw<SlipLaw<TPMethod>, TPMethod>::SlowVelocityWeakeningLaw;
+  using SlowVelocityWeakeningLaw<SlipLaw<TPMethod>, TPMethod>::copyLtsTreeToLocal;
+
+  double updateStateVariable(int pointIndex,
+                             unsigned int face,
+                             double stateVarReference,
+                             double timeIncrement,
+                             double localSlipRate) {
+    const double localSl0 = this->sl0[face][pointIndex];
+    const double exp1 = std::exp(-localSlipRate * (timeIncrement / localSl0));
+    return localSl0 / localSlipRate * std::pow(localSlipRate * stateVarReference / localSl0, exp1);
+  }
```

Comment:
Why do we need this method? Isn't the `Impl` version below always used?

Reply:
Yep, you're 100% right. It was an artifact during my development. I just copied it from the CPU version to have it in font of me and I forgot to delete it.

Thanks a lot for noticing!